api/credentials/extensions/repositories/vault/auth.go

```diff
@@ -51,7 +51,7 @@ func (r *AuthMethods) Names() []string {
 	r.lock.Lock()
 	defer r.lock.Unlock()
 
-	names := maps.Keys(r.methods)
+	names := utils.MapKeys(r.methods)
```

Comment:
```suggestion
	names := slices.Collect(maps.Keys(r.methods))
```
I guess that would be the new way to handle this using the standard library

Reply:
https://github.com/open-component-model/ocm/pull/1308